internal/uploader.go

```diff
@@ -216,3 +227,32 @@ func (uploader *SplitStreamUploader) Clone() Uploader {
 		blockSize:  uploader.blockSize,
 	}
 }
+
+func (uploader *RegularUploader) ShowRemainingTime() {
+	defer uploader.waitGroup.Done()
+	startTime := time.Now()
+	for {
+		compressedSizeInt, err := uploader.UploadedDataSize()
+		if err != nil {
+			return
+		}
+		uncompressedSizeInt, err := uploader.RawDataSize()
+		if err != nil {
+			return
+		}
+
+		compressedSize := float64(compressedSizeInt) / 1000000
+		uncompressedSize := float64(uncompressedSizeInt) / 1000000
+
+		timeElapsed := time.Since(startTime).Seconds()
+		uploader.compressedUploadSpeeds.Add(compressedSize / timeElapsed)
+		uploader.uncompressedUploadSpeeds.Add(uncompressedSize / timeElapsed)
+
+		tracelog.InfoLogger.Printf("Uploaded compressed: %v Mb\n", compressedSize)
```

Comment:
These 4 messages might bloat the logs a little, lets pack them into a single upload statistics message, e.g.:
`Uploaded: 124MB, average 10MB/s. Raw data read: 339MB, average 34MB/s`.